lib/bin/supllib/license.txt

```diff
@@ -0,0 +1,70 @@
+Copyright (c) 2019 - 2019, Nordic Semiconductor ASA
+All rights reserved.
+
+1.  By downloading this software, you acknowledge that all right, title and interest in and to the
```

Comment:
Again, do we need this license here? The file itself (the library) is not here, so I am not sure it makes sense at all. The license should be displayed and shown when downloading the library from Nordic's website, but not included here? cc @bjorn-tore-taraldsen

Reply:
I agree that including the license here is a bit strange as there is nothing to license here.

Reply:
We could be okay with only having the license in the .h file, but we need to visualize for the user that this library comes with a different license.